src/globals/scss/_helper-mixins.scss

```diff
@@ -56,6 +56,11 @@
   }
 }
 
+@mixin focus-outline--x {
+  outline: 2px solid $brand-01;
+  outline-offset: -2px;
+}
+
```

Comment:
Should we add this to this mixin and just re-name it to outline? I had added a `$focus` variable for the color (should maybe be `$focus-color`?) 

```
@mixin focus-outline($type: 'border') {
  @if ($type == 'border') {
    outline: 1px solid $brand-01;
  }

  @if ($type == 'blurred') {
    box-shadow: 0 0 0 3px $focus;
    outline: 1px solid transparent;
  }

  @if ($type == 'outline') {
    outline: 2px solid $focus;
    outline-offset: -2px;
  }
}
```

Or I'm totally fine w/ leaving it for now and just making sure we do a pass on all the components when we are done to try and consolidate how we are doing focus states. 😄